kartograf/merge.py

```diff
@@ -1,20 +1,52 @@
 import ipaddress
 import shutil
-import numpy as np
-from pandarallel import pandarallel
 import pandas as pd
 
 from kartograf.timed import timed
 
+class Networks:
+    '''
+    A class whose _dict represents a mapping of the network number and IP networks within that network for a given AS file.
+
+    To check inclusion of a given IP network in the base AS file, we can compare (see check_inclusion) the networks under the root network number instead of all the networks in the base file.
+    '''
+    def __init__(self):
+        self._dict = dict()
+        self._keys = self._dict.keys()
```

Comment:
I don't think the root_nets could overlap since ipv6 are 16 bit numbers and we're just taking the string of the octet here? Besides, it's iterating through the dictionary entry for that octet that affects speed, not necessarily finding the entry. In the asmap.py [code](https://github.com/bitcoin/bitcoin/blob/6463117a29294f6ddc9fafecfd1e9023956cc41b/contrib/asmap/asmap.py#L18), they remap all IPv4 to IPv6, which I considered but it wouldn't work with the dictionary approach.